test/evergreen.yml

```diff
@@ -272,16 +272,26 @@ functions:
 
             echo "Checking out branch $branch ..."
             git checkout $branch
-            sh reconf
 
             # Java API is removed in newer branches via WT-6675.
             if [ $branch == "mongodb-4.2" ] || [ $branch == "mongodb-4.0" ]; then
+              cd build_posix
+              sh reconf
               ../configure CFLAGS="-DMIGHT_NOT_RUN -Wno-error" --enable-java --enable-python --enable-strict
               (cd lang/python && make ../../../lang/python/wiredtiger_wrap.c)
               (cd lang/java && make ../../../lang/java/wiredtiger_wrap.c)
-            else
+            elif [ $branch == "mongodb-5.0" ] || [ $branch == "mongodb-4.4" ]; then
+              cd build_posix
+              sh reconf
               ../configure CFLAGS="-DMIGHT_NOT_RUN -Wno-error" --enable-python --enable-strict
               (cd lang/python && make ../../../lang/python/wiredtiger_wrap.c)
+            else
+              . test/evergreen/find_cmake.sh
+              if [ -d cmake_build ]; then rm -r cmake_build; fi
+              mkdir -p cmake_build
+              cd cmake_build
+              $CMAKE -DCMAKE_C_FLAGS="-DMIGHT_NOT_RUN -Wno-error" -DENABLE_PYTHON=1 -DENABLE_STRICT=1 -G "${cmake_generator|Ninja}" ../.
+              ninja lang/python/all
```

Comment:
Question here, what does `lang/python/all` do? Is it the equivalent to `make ../../../lang/python/wiredtiger_wrap.c`?

Reply:
Its not quite equivalent, `lang/python/all` will compile the full Python API. Unfortunately the CMake build doesn't have a target at the same granularity that just generates `wiredtiger_wrap.c`. `lang/python/all` is the closest here.